### DISCUSSION TITLE
Is this select check in MRT_EditCellTextField correct?

### DISCUSSION THREAD

**Discussion by loughlinclaus3** [KevinVandy/material-react-table#948] [Q&A] [answered]
So, in the [MRT_EditCellTextField, on line 74, there's an `if (textFieldProps?.select)` check](https://github.com/KevinVandy/material-react-table/blob/v2/packages/material-react-table/src/components/inputs/MRT_EditCellTextField.tsx#L74) that seems feels like a mistake to me. It's checking that the user has set an `muiEditTextFieldProps` value of `select: true` in the column options, but shouldn't that be determined by the `editVariant: "select"` column option?

A user would need to override the variant check of line 124 with the select in textFieldProps on line 128, which while a totally valid and functional approach, feels kind of redundant and seemingly removes the purpose of declaring the editVariant at all.

**Comment by KevinVandy** [comment]
yeah maybe. Did you notice any present bugs?

**Reply by loughlinclaus3** [reply, marked as answer]
~~It's not causing any bugs no, but whatever the original purpose of that if statement was, it is likely no longer being handled.~~ Actually, just noticed that you've since updated the code to handle this, thanks!
